platform_versioned_docs/version-23.3.0/enterprise/kubernetes.mdx

```diff
@@ -68,82 +71,82 @@ Seqera Platform Enterprise is distributed as a collection of Docker containers a
     ```
 
 :::note
-The `configmap.yml` manifest includes both the `tower.env` file and `tower.yml` file. These files are made available to the other containers through volume mounts.
+The `configmap.yml` manifest includes both the `tower.env` and `tower.yml` files. These files are made available to the other containers through volume mounts.
```

Comment:
The only thing I would recommend here is to consolidate items 1 and 2 under "Seqera ConfigMap" into two sentences without numbering, unless you can find a third item.